packages/visx-zoom/src/Zoom.tsx

```diff
@@ -79,229 +80,267 @@ type ZoomState = {
   isDragging: boolean;
 };
 
-class Zoom extends React.Component<ZoomProps, ZoomState> {
-  static defaultProps = {
-    passive: false,
-    scaleXMin: 0,
-    scaleXMax: Infinity,
-    scaleYMin: 0,
-    scaleYMax: Infinity,
-    transformMatrix: {
-      scaleX: 1,
-      scaleY: 1,
-      translateX: 0,
-      translateY: 0,
-      skewX: 0,
-      skewY: 0,
-    },
-    wheelDelta: (event: React.WheelEvent | WheelEvent) =>
-      -event.deltaY > 0 ? { scaleX: 1.1, scaleY: 1.1 } : { scaleX: 0.9, scaleY: 0.9 },
-    style: undefined,
-    className: undefined,
-  };
-
-  containerRef: HTMLDivElement | null = null;
-
-  startPoint: Point | undefined = undefined;
+function Zoom<ElementType extends Element>({
+  scaleXMin = 0,
+  scaleXMax = Infinity,
+  scaleYMin = 0,
+  scaleYMax = Infinity,
+  initialTransformMatrix = {
```

Comment:
just looking for optimization opportunities, could we pull this out into a constant `defaultInitialTransformMatrix`? I think this way of defining it will be a new object each render (which doesn't matter for `useRef`/`useState` but I think will break memoization of a callback below)